src/web/mjs/connectors/Batoto.mjs

```diff
@@ -32,11 +32,18 @@ export default class Batoto extends AnyACG {
         return this.config.url.value;
     }
 
+    set url(value) {
+        if (this.config && value) {
```

Comment:
Leave implementation blank, otherwise the user defined value will be unintentionally overwritten by the application

Reply:
This was copied from other implementations of this